src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -64,6 +73,22 @@ internal void AddFunction(TexlFunction function)
 
             _extraFunctions.Add(function.GetUniqueTexlRuntimeName(), function);
         }
+                
+        internal bool TryGetSymbol(DName name, out IExternalEntity symbol, out DName displayName)
+        {
+            var lookupName = name;
+            if (_environmentSymbolDisplayNameProvider.TryGetLogicalName(name, out var logicalName))
+            {
+                lookupName = logicalName;
+                displayName = name;
+            }
+            else if (_environmentSymbolDisplayNameProvider.TryGetDisplayName(name, out displayName))
```

Comment:
>TryGetDisplayName [](http://example.com/codeflow?start=59&length=17)

should lookup by logical name be done first to have higher precedence?

Reply:
Good point. I'll also update it to include entities without display names in the collision checks, so lookups by one vs the other won't have any shadowing/precedence behavior.